components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1154,6 +1159,40 @@ interface AssetRatioService {
   GetCoinMarkets(string vs_asset, uint8 limit) => (bool success, array<CoinMarket> values);
 };
 
+interface MeldIntegrationService {
+  // Obtains the list of Service Providers
+  // All parameters are optional
+  GetServiceProviders(MeldFilter filter)
+    => (array<MeldServiceProvider>? service_providers, array<string>? error);
+
+  // Obtains the crypto quotes
+  // Note that next parameters are mandatory:
+  // destination_currency_code, country, source_amount, source_currency_code
+  GetCryptoQuotes(string country, string source_currency_code,
+    string destination_currency_code, double source_amount, string? account)
+    => (array<MeldCryptoQuote>? crypto_quotes, array<string>? error);
+
+  // Obtains the list of payment methods
+  // All parameters are optional
+  GetPaymentMethods(MeldFilter filter)
```

Comment:
When calling this we need to make sure we're rendering the images in chrome-untrusted frames, but that's to be handled in other PRs.